packages/components/organisms/f-header/src/assets/scss/common.scss

```diff
@@ -42,3 +42,10 @@ $header--transparent-opacity        : 0.7;
         padding-top: 0;
     }
 }
+
+// Hides element in-between the mid and wide breakpoint
+.hide-on-mid {
+    @include f.media('>mid', '<wide') {
+        display: none !important;
```

Comment:
All LGTM - MY only question, Is this `!important` needed?

Reply:
does this kind of class warrant being added as a global utility class here?
https://github.com/justeat/fozzie-components/blob/master/packages/tools/fozzie/src/scss/trumps/_rwd.scss

Reply:
That's fair! TY for explanation :)

Reply:
> does this kind of class warrant being added as a global utility class here? https://github.com/justeat/fozzie-components/blob/master/packages/tools/fozzie/src/scss/trumps/_rwd.scss

Yea, that could be useful! 😄 @jamieomaguire

Reply:
Could you get away with just using a show/hide statement inside vue to display what you need instead? i.e `v-show="isCondensed"`

Reply:
Unfortuatley no, as that would hide the icons on all breakpoints, we only want to hide the icons on the mid breakpoint. I also want to stay away from getting the screen size through JS as that would cause the icons to flash on load while the JS is still loading 😄